module/VuFind/src/VuFind/Content/Covers/Koha.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Koha cover content loader.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\Content\Covers;
+
+/**
+ * Koha cover content loader.
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:content_provider_components
+ */
+class Koha extends \VuFind\Content\AbstractCover
+{
+    /**
+     * Base URL for Koha covers
+     *
+     * @var string
+     */
+    protected $url;
+
+    /**
+     * Constructor
+     *
+     * @param string $url Base URL for Koha covers
+     */
+    public function __construct(string $url)
+    {
+        $this->url = $url;
```

Comment:
I needed to add this to get it working (and state an expectation that anybody who is using Koha's local covers feature probably prefers that the results get cached.

```
$this->supportsRecordid = true;
$this->cacheAllowed = true;
```